ci/jobs/projects.yaml

```diff
@@ -155,3 +155,15 @@
             - f23-np
             - rhel6-np
             - rhel7-np
+
+- project:
+    name: satellite6-pulp-automation
+    os:
+      - rhel7
+    pulp_version: 2.8.3
+    upgrade_pulp_version:
+        - 2.8.6
+    jobs:
+        - '{name}'
```

Comment:
Better to translate this variable, use `satellite6-pulp-automation` here to make easy to read and understand by newcomers.